tests/posix/common/src/timer.c

```diff
@@ -104,7 +104,7 @@ ZTEST(timer, test_CLOCK_MONOTONIC__SIGEV_THREAD)
 
 ZTEST(timer, test_timer_overrun)
 {
-	struct sigevent sig = { 0 };
+	struct sigevent sig = {0};
```

Comment:
sorry, the title of this commit 7ac39644153742f42b95025d9fd4d93433996267 should be something like `test: posix: timer: fix format` to reflect the 'area' as noted in the contribution guideline

Reply:
just also note that this change might conflict with the relocation #80670

Reply:
> > style: for code style changes
> 
> in the contribution guideline. I though it was for this kind of commit.

I'd use that title if the commit modified more than a few areas. Your commit only affected one file, it is more informative to have a specific area, so that we know what this commit does when scrolling through the git log, without having to click into it.